third_party/amd/lib/TritonAMDGPUToLLVM/DotOpToLLVM/WMMA.cpp

```diff
@@ -227,39 +226,60 @@ std::string addInstructionSuffix(std::string intrinsicName, unsigned kWidth,
 }
 
 Value generateWMMAIntrinsic(ConversionPatternRewriter &rewriter, Location loc,
-                            Value valA, Value valB, Value valC, Type aElType,
-                            Type bElType, Type dElType, StringRef name,
-                            std::optional<bool> tiedLower) {
+                            int wmmaVer, Value valA, Value valB, Value valC,
```

Comment:
Can we add some lit tests to check wmma v3 and the emitted amdgcn intrinsics?

Reply:
sure working on it now